build.gradle.kts

```diff
@@ -21,6 +25,14 @@ repositories {
     mavenCentral()
 }
 
+plugins {
+    id("org.jetbrains.dokka") version "1.6.10"
```

Comment:
I tried, it needs to be defined in multiple blocks for some reason

Reply:
The `buildScript` block is very special. 
It's the very first thing that Gradle will run and it defines the dependencies for the rest of the file itself and other files.

So we can't define something inside of it, or make it see something that is defined outside of it - because whatever is outside didn't run yet.

Reply:
Not doable